tox.ini

```diff
@@ -1,8 +1,16 @@
 [tox]
-envlist = pypy, pypy3, py27, py34, py35, py36
+envlist = pypy, pypy3, py27, py34, py35, py36, flake8
 
 [testenv]
 deps = -rrequirements-dev.txt
 commands =
     py.test sasstests.py
+
+[testenv:flake8]
+deps= flake8
+skip_install=True
+commands=
     flake8 .
```

Comment:
So? `flake8` should complain about python 3-only syntax because this project still is compatible with it. Since you dropped 3.3 I'm not aware of any syntax that's valid in 2.7 and not in 3.4+ (`u''` comes to mind for 3.3).